pkg/kcp/provider/gcp/vpcpeering/loadKymaVpcPeering.go

```diff
@@ -0,0 +1,45 @@
+package vpcpeering
+
+import (
+	"context"
+	"fmt"
+	"github.com/kyma-project/cloud-manager/api/cloud-control/v1beta1"
+	"github.com/kyma-project/cloud-manager/pkg/composed"
+	"github.com/kyma-project/cloud-manager/pkg/util"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func loadKymaVpcPeering(ctx context.Context, st composed.State) (error, context.Context) {
+	state := st.(*State)
+	logger := composed.LoggerFromCtx(ctx)
+
+	if state.kymaVpcPeering != nil {
+		return nil, ctx
```

Comment:
not a big deal but you can return nil, nil since ctx is not modified and avoid unnecessary ctx assignmet in 
```
# pkg/composed/action.go
if nextCtx != nil {
	currentCtx = nextCtx
}
```